ts/features/wallet/onboarding/bancomat/screens/search/SearchBankComponent.tsx

```diff
@@ -97,6 +97,7 @@ export const SearchBankComponent: React.FunctionComponent<Props> = (
           data={filteredList}
           renderItem={renderListItem(true)}
           keyExtractor={keyExtractor}
+          keyboardShouldPersistTaps={"always"}
```

Comment:
This PR lgtm but imho as an improvement, we should dismiss the keyboard when we tap on the selected item

So I would change as suggested this prop value
```suggestion
          keyboardShouldPersistTaps={"handled"}
```

and add an additional `Keyboard.dismiss()` on item onPress function

what do you think?